package.json

```diff
@@ -25,7 +26,7 @@
   "files": [
     "build",
     "package.json",
-    "yarn.lock"
+    "package-lock.json"
```

Comment:
I don't think this is actually neccessary. `npm` will ignore package-lock.json from being published